.github/workflows/fiddle.yml

```diff
@@ -37,6 +37,8 @@ jobs:
         run:  rake install
 
       - name: rake test
+        env:
+          rubyopt: --disable=gems
```

Comment:
Why is this `rubyopt` not `RUBYOPT`?

Reply:
I can change it.  Windows ENV 'keys' are case-insensitive.  So, it only affects Windows builds.

Reply:
OK. Could you use `RUBYOPT` to avoid confusion?

Reply:
Yes and done.